### PR TITLE
Fix az CI

### PR DESCRIPTION
**Pull request by jim-minter**
fix CI az at latest 2.0.z (2.0.81) for now
    
rationale: >= 2.1.0 deprecates python 2.7 support, and for now it makes sense for us to keep testing that our extension works on older az releases and 2.7.
    
perhaps later we should test against multiple az releases (2.0.latest, 2.2.latest, etc.))

### CODE REVIEW

**Comment by jim-minter** [comment]
@julienstroheker wdyt?

File: hack/devtools/setup-azure-tools.sh [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
-curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
+# fixing to latest az 2.0.z for now to continue testing 2.7, 3.5 compatibility
+# curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+sudo apt-get update
+sudo apt-get install -y apt-transport-https lsb-release gnupg curl
+sudo bash -c 'curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor >/etc/apt/trusted.gpg.d/microsoft.asc.gpg'
+sudo bash -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $(lsb_release -cs) main" >/etc/apt/sources.list.d/azure-cli.list'
+sudo apt-get update
+sudo apt-get install -y azure-cli=2.0.81
```
Comment (julienstroheker) [inline hack/devtools/setup-azure-tools.sh]: `apt-get install -y azure-cli=2.0.81-1~$(lsb_release -cs)`

Comment (julienstroheker) [reply hack/devtools/setup-azure-tools.sh]: I added this at the same time you pushed your update :) nervermind

**Comment by jim-minter** [comment]
@JackQuincy this should also fix the az version issue you mentioned in your demo

**Pull request merged**